bip-0340/reference.py

```diff
@@ -26,13 +26,15 @@ def tagged_hash(tag: str, msg: bytes) -> bytes:
     tag_hash = hashlib.sha256(tag.encode()).digest()
     return hashlib.sha256(tag_hash + tag_hash + msg).digest()
 
-def is_infinity(P: Optional[Point]) -> bool:
+def is_infinite(P: Optional[Point]) -> bool:
     return P is None
 
 def x(P: Point) -> int:
+    assert not is_infinite(P)
```

Comment:
nit: this appears to be redundant because the `Point` type  already ensures that this is not the point at infinity

Reply:
Maybe it's still useful for documentation purposes? Not everyone may be familiar with the typing semantics, and they only get checked if you run it with `mypy`.

Reply:
Yes, I'd leave it there. I guess good asserts are always redundant, right?

Reply:
Okay, ACK